packages/react-components/src/components/HorizontalGallery.tsx

```diff
@@ -76,7 +76,7 @@ export const HorizontalGallery = (props: HorizontalGalleryProps): JSX.Element =>
   const clippedPage = firstIndexOfCurrentPage < numberOfChildren - 1 ? page : lastPage;
 
   const childrenOnCurrentPage = useMemo(() => {
-    if (indexesArray[0] !== undefined) {
+    if (indexesArray && indexesArray[clippedPage] !== undefined) {
```

Comment:
Just wondering if it is possible to get indexesArray[clippedPage] to be undefined(without my change), since the logic of clippedPage here seems no issue (no need to find answer, just looking for ideas) @mgamis-msft